src/Common/src/System/Security/Cryptography/RSAOpenSsl.cs

```diff
@@ -58,10 +59,28 @@ public override int KeySize
             }
         }
 
+        private void SetKeySize(int newKeySize)
```

Comment:
The name "SetKeySize()" glosses over its true purpose. "ForceKeySize()" would better distinguish it from setting the KeySize property in the eyes of the casual reader. Copying over the explanatory comment from the desktop version of this would be good so people know we didn't just do this out of paranoia.

Otherwise, LGTM